specification/ai/data-plane/HealthInsights/stable/2024-04-01/openapi.json

```diff
@@ -37,9 +37,24 @@
   "security": [
     {
       "AzureKey": []
+    },
+    {
+      "AADToken": [
+        "https://cognitiveservices.azure.com/.default"
+      ]
     }
   ],
   "securityDefinitions": {
+    "AADToken": {
+      "type": "oauth2",
+      "description": "The Azure Active Directory OAuth2 Flow",
```

Comment:
Azure Active Directory is now Microsoft Entra ID https://learn.microsoft.com/en-us/entra/fundamentals/how-to-rename-azure-ad

Reply:
@vicancy .
Is this what you meant ?